README.markdown

```diff
@@ -8,8 +8,7 @@ nodes, and view inventory data and backed-up file contents.
 Dependencies
 ------------
 
-* Ruby 1.8.7, 1.9.3, 2.0.0 or 2.1.x
-* Bundler >= 1.1
+* Ruby 2.2, 2.3, 2.4, 2.5
```

Comment:
should this mention puppet versions supported as well?

Reply:
AFIAK we simply support every version in existence until now :smile: so maybe we can just add up to what version we have tested it or something like that. What do you think?

Reply:
we can do this separately